meetings/2021-01-14.md

```diff
@@ -0,0 +1,43 @@
+# 2021-01-1478 4th meeting
+
+## Attendees
+
+* [Ben Adams](https://github.com/benaadams)
+* [Claire Novotny](https://github.com/clairernovotny)
+* [Eric Erhardt](https://github.com/eerhardt)
+* [Immo Landwerth](https://github.com/terrajobst)
+* [Jeremy Sinclair](https://github.com/snickler)
+* [John Callaway](https://github.com/ovation22)
+* [Ken Cutt](https://github.com/kencutt)
+* [Sébastien Ros](https://github.com/sebastienros)
+
+## Talking points
+
+* Discussed the proposed [survey](../docs/survey.md)
+    - The phrase "level playing field" is hard to understand
+    - We should add a similar question on trust for other big names, such as
+      Amazon, Google, component vendors
+    - We should add a question on trust regarding "open source" vs "paid source"
+    - We should add a question around money/funding for maintainers
+    - We should add question around willingness to pay for a library
```

Comment:
How about adding the same question, but in terms of contributing back instead of paying?

Reply:
Good call!